main/nav_backplane_viewer.py

```
@@ -0,0 +1,1366 @@
+        try:
+            if self._current_image_name:
+                default_name = f'{self._current_image_name}_backplanes.png'
+        except Exception:
+            pass
+        path, _ = QFileDialog.getSaveFileName(self, 'Save PNG', default_name, 'PNG Files (*.png)')
+        if not path:
+            return
+        try:
+            qimg.save(path, 'PNG')
+        except Exception:
+            pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)